Examples/other_plugins/UIKitScreenTracking.swift

```diff
@@ -122,7 +122,7 @@ extension UIViewController {
         guard let top = Self.seg__visibleViewController(activeController()) else { return }
         
         var name = String(describing: top.self.classForCoder).replacingOccurrences(of: "ViewController", with: "")
-        print(name)
+        analytics?.log(message: name)
```

Comment:
this should probably be more descriptive than just the name of the view.  ie: "Auto-tracking Screen: \(name)"